components/finance/ExpenseButton/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, createStyles, makeStyles } from '@material-ui/core'
+import React, { useState } from 'react'
+import { useToggle } from 'react-use'
+
+import ClientSideRendering from '../../../lib/client-side-rendering'
+import ExpensePanelDetails from '../ExpensePanelDetails'
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    deleteButton: {
+      color: 'white',
+      background: '#f16c6c',
+      '&:hover': {
+        background: '#f16c6c',
+      },
+    },
+  })
+)
+
+export default function ExpenseButton() {
```

Comment:
Call the component IncomeExpenseBrick as it will be used for income type as well